requirements.txt

```diff
@@ -1,2 +1,3 @@
 mozilla-schema-generator
 stringcase
+flake8
```

Comment:
`black` at least (along with `flake8-black`, see above) and `isort` should be added to this list.